partner_personal_information_access/views/res_partner_forms.xml

```diff
@@ -83,70 +118,59 @@
+            <xpath expr="//label[@for='email']" position="after">
+                <field
+                    name="email"
+                    invisible="1"
+                    groups="!base.group_private_addresses"
+                />
+            </xpath>
+            <xpath
+                expr="//field[@name='email'][@widget='email']/.."
+                position="attributes"
+            >
                 <attribute name="groups">base.group_private_addresses</attribute>
-            </field>
+            </xpath>
         </field>
     </record>
 
-    <record id="partner_form_send_sms_form_view" model="ir.ui.view">
-        <field name="name">res.partner.form.send.sms</field>
+    <record id="res_partner_view_form" model="ir.ui.view">
+        <field name="name">res.partner.view.form.inherit.sms</field>
         <field name="model">res.partner</field>
-        <field name="inherit_id" ref="sms.partner_form_send_sms_form_view" />
+        <field name="inherit_id" ref="sms.res_partner_view_form" />
         <field name="arch" type="xml">
             <xpath expr="//label[@for='phone']" position="attributes">
                 <attribute name="groups">base.group_private_addresses</attribute>
             </xpath>
-            <xpath expr="//field[@name='phone']/.." position="attributes">
-                <attribute name="groups">base.group_private_addresses</attribute>
-            </xpath>
             <xpath expr="//label[@for='mobile']" position="attributes">
                 <attribute name="groups">base.group_private_addresses</attribute>
             </xpath>
-            <xpath expr="//field[@name='mobile']/.." position="attributes">
-                <attribute name="groups">base.group_private_addresses</attribute>
+            <xpath expr="//label[@for='phone']" position="after">
+                <field
+                    name="phone"
+                    invisible="1"
+                    groups="!base.group_private_addresses"
+                />
             </xpath>
-        </field>
-    </record>
-
-    <record id="view_partner_short_form" model="ir.ui.view">
-        <field name="name">res.partner.form</field>
-        <field name="model">res.partner</field>
-        <field name="inherit_id" ref="base.view_partner_short_form" />
-        <field name="arch" type="xml">
-            <xpath expr="//label[@for='street']" position="attributes">
-                <attribute name="groups">base.group_private_addresses</attribute>
+            <xpath expr="//label[@for='mobile']" position="after">
+                <field
+                    name="mobile"
+                    invisible="1"
+                    groups="!base.group_private_addresses"
+                />
             </xpath>
-            <xpath expr="//div[hasclass('o_address_format')]" position="attributes">
+            <xpath
+                expr="//field[@name='phone'][@widget='phone']/.."
+                position="attributes"
+            >
```

Comment:
Is there a reason why you target with `@name` and `@widget` and not just with `<field name="phone" position="attributes">` ?